R/centrality.R

```diff
@@ -1390,10 +1404,33 @@ eigen_centrality <- function(
     }
   }
 
+  mode <- igraph.match.arg(mode)
+
+  if (lifecycle::is_present(directed)) {
+    if (directed) {
+      lifecycle::deprecate_soft(
+        "2.2.0",
```

Comment:
Here and everywhere:

```suggestion
        "3.0.0",
```